Cargo.toml

```
@@ -59,3 +60,8 @@ harness = false
 [[bench]]
 name = "point_bench"
 harness = false
+
+[patch.crates-io]
+bls12_381 = { git = "https://github.com/nuttycom/bls12_381.git", branch = "bitvec-0.22", default-features = false }
```

Once https://github.com/zkcrypto/bls12_381/pull/62 merges:
```suggestion
bls12_381 = { git = "https://github.com/zkcrypto/bls12_381.git", branch = "main", default-features = false }
```